features/other_b/skin_public.feature

```diff
@@ -120,7 +120,8 @@ Feature: Public skins
     And I follow "Preview"
   Then I should be on the works tagged "Major Crimes"
     And I should see "You are previewing the skin Usable Skin. This is a randomly chosen page."
-    And I should see "Go back or click any link to remove the skin"
-    And I should see "Tip: You can preview any archive page you want by tacking on '?site_skin=[skin_id]' like you can see in the url above."
-  When I follow "Return To Skin To Use"
+    And I should see "Go back or follow any link to remove the skin"
+    And I should see "Tip: You can preview any archive page you want by adding '?site_skin="
+    And I should see "' to the end of the URL."
```

Comment:
I'm not familiar enough with Ruby test suites to have been able to figure out how to put the actual test-skin ID into this string, so split it up into the before and after parts. If there's a better way to do that I would love to learn :)

Reply:
I think this is fine; if we wanted to check the actual ID, we would need to add a custom step to https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/skin_steps.rb (which IMO is overkill here)